engine-rest/engine-rest-jakarta/pom.xml

```diff
@@ -0,0 +1,646 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>camunda-engine-rest-core-jakarta</artifactId>
+  <name>Camunda Platform - engine - REST Jakarta</name>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-engine-rest-root</artifactId>
+    <relativePath>../</relativePath>
+    <version>7.19.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <rest.http.port>38081</rest.http.port>
+    <version.resteasy>6.2.2.Final</version.resteasy>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>jakarta.platform</groupId>
+        <artifactId>jakarta.jakartaee-bom</artifactId>
+        <version>${version.jakarta-ee-spec}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-bom</artifactId>
+        <version>${version.resteasy}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>commons-fileupload</groupId>
+      <artifactId>commons-fileupload</artifactId>
+    </dependency>
+    <!-- override managed version from commons-fileupload -->
```

Comment:
🙃 Shouldn't this comment go above the dependency?

Reply:
Well, it should. Copy paste from `engine-rest` :)

Reply:
It actually overrides the `commons-io` version which is also defined in the `commons-fileupload`. So I think it's at the right place.

Reply:
Can we additionally exclude `commons-io` from `commons-fileupload`? That makes it more clear.